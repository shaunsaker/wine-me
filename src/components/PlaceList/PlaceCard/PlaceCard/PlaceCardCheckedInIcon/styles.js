import { StyleSheet } from 'react-native';

import styleConstants from '../../../../../styleConstants';

const SIZE = 30;

const styles = StyleSheet.create({
  container: {
    width: SIZE,
    height: SIZE,
    borderRadius: SIZE / 2,
    backgroundColor: styleConstants.colors.white,
    ...styleConstants.shadows.small,
    justifyContent: 'center',
    alignItems: 'center',
  },
  icon: {
    fontSize: styleConstants.fonts.sizes.icon,
    color: styleConstants.colors.success,
  },
});

export default styles;
