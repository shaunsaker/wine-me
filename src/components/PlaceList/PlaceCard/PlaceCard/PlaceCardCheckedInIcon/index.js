import React from 'react';
import { View } from 'react-native';
import Icon from 'react-native-vector-icons/MaterialIcons';

import styles from './styles';

const propTypes = {};

const defaultProps = {};

const PlaceCardCheckedInIcon = () => {
  return (
    <View style={styles.container}>
      <Icon name="check" style={styles.icon} />
    </View>
  );
};

PlaceCardCheckedInIcon.propTypes = propTypes;
PlaceCardCheckedInIcon.defaultProps = defaultProps;

export default PlaceCardCheckedInIcon;
