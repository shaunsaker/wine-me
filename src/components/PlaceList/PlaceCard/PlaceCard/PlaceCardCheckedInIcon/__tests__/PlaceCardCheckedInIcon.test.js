import React from 'react';
import renderer from 'react-test-renderer';

import PlaceCardCheckedInIcon from '..';

describe('PlaceCardCheckedInIcon', () => {
  describe('renders', () => {
    it('renders', () => {
      const component = renderer.create(<PlaceCardCheckedInIcon />);

      expect(component).toMatchSnapshot();
    });
  });
});
