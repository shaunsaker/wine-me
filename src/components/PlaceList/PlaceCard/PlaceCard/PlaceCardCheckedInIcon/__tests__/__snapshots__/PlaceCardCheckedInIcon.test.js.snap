// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceCardCheckedInIcon renders renders 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderColor": null,
      "borderRadius": 15,
      "borderWidth": 0,
      "elevation": 2,
      "height": 30,
      "justifyContent": "center",
      "shadowColor": "#212121",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.17,
      "shadowRadius": 2,
      "width": 30,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#4caf50",
          "fontSize": 24,
        },
        Object {
          "fontFamily": "Material Icons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    î—Š
  </Text>
</View>
`;
